README.md

```diff
@@ -52,15 +57,15 @@ end
 
 You can then start synchronizing files with S3 through ```middleman s3_sync```.
 
-### Configuration Defaults
+#### Configuration Defaults
 
 The following defaults apply to the configuration items:
 
 | Setting                    | Default                            |
 | -----------------          | ----------------------------       |
-| aws_access_key_id          | ```ENV['AWS_ACCESS_KEY_ID']```     |
-| aws_secret_access_key      | ```ENV['AWS_SECRET_ACCESS_KEY']``` |
-| bucket                     | ```ENV['AWS_BUCKET']```            |
+| aws_access_key_id          | -                                  |
```

Comment:
The default value of the configuration (without doing anything) is nil. ENV vars and Thor args are documented separately.

Side note: I could not get the ENV vars to work from command line. How does one pass in ENV vars with Thor?